### PR TITLE
Upgrade to Solr 8.8.1

### PR DESCRIPTION
**Pull request by frnkshin**
#### What does this PR do?
Upgrades to Solr 8.8.1

#### Who is reviewing it? 

@jMoneee
@shaundmorris
@abel-connexta
@kentmorrissey
@lavoywj
@frnkshin

#### Select relevant component teams: 
@codice/solr 

#### Ask 2 committers to review/merge the PR and tag them here.
@pklinef
@lambeaux

#### How should this be tested?

```
1.  Run SolrCloud docker image as directed by https://github.com/codice/ddf/blob/master/distribution/docker/solrcloud/README.md
2. Verify version by going into Solr admin console
3. Index data and verify search works
```

#### Any background context you want to provide?
Upgrade Solr to 8.8.1
https://lucene.apache.org/solr/8_8_1/changes/Changes.html

#### What are the relevant tickets?
Fixes: #n/a

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by frnkshin** [comment]
build now

File: pom.xml [hunk 1]
```diff
@@ -290,8 +290,8 @@
         <servicemix.specs.version>2.9.0</servicemix.specs.version>
         <servlet-api.version>3.1.0</servlet-api.version>
         <smtp.client.version>1.7.1</smtp.client.version>
-        <solr.version>8.7.0</solr.version>
-        <solr.docs.version>8_7</solr.docs.version>
+        <solr.version>8.8.0</solr.version>
```
Comment (pklinef) [inline pom.xml:293]: Solr 8.8.1 just recently released with 5 bug fixes.

https://lucene.apache.org/solr/news.html#apache-solrtm-881-available

**Review by pklinef — CHANGES_REQUESTED** [review]

**Comment by frnkshin** [comment]
build now

**Review by shaundmorris — APPROVED** [review]

**Comment by frnkshin** [comment]
build now

**Comment by frnkshin** [comment]
build now

**Review by shaundmorris — APPROVED** [review]
Do we need to specific Solr cloud tests as well?

**Review by lavoywj — APPROVED** [review]

**Comment by pklinef** [comment]
Solr 8.8.1 has moved to Commons IO 2.8.0.

https://github.com/codice/ddf/blob/7dac6e1c4b78a3f156b30c964c9bac24b4d86939/platform/solr/solr-dependencies/pom.xml#L73-L77

**Review by AzGoalie — APPROVED** [review]

**Comment by frnkshin** [comment]
build now

**Review by pklinef — APPROVED** [review]

**Pull request merged**